packages/obonode/obojobo-sections-assessment/server/express.test.js

```diff
@@ -193,9 +193,12 @@ describe('server/express', () => {
 	test('POST /api/assessments/attempt/mock-attempt-id/resume errors', async () => {
 		expect.hasAssertions()
 		const mockReturnValue = {}
+		const errorString = 'Cannot resume an attempt for a different module'
```

Comment:
There's a lot of inline constants, possible to move these out into their own file that can be shared between the different assessment server files?

Reply:
That'd definitely simplify things! Do you have any suggestions on where I could move these?